docs/embedding/modular-embedding.md

```diff
@@ -152,70 +159,24 @@ When you're creating a new embed using **Admin > Embedding > Setup guide > Embed
 
 - **Show title**: what it says on the tin.
 
-- **Allow editing dashboards and questions**: lets people create and edit dashboards or questions in the current collection. When disabled, they can still perform actions like filter, summarize, and drill-through, but won’t be able to save results.
+- **Allow editing dashboards and questions**: lets people create and edit dashboards or questions in the current collection. When disabled, they can still perform actions like filter, summarize, and drill-through, but won't be able to save results.
 
-## Configuring embeds
+## Page embedding settings
```

Comment:
This doesn't parse well for me. Maybe page-level  or something? TBH i'd prefer to keep config somewhere in the section title because it maps to what the correponding part of the snippet is called